tools/mo/openvino/tools/mo/serialize.py

```diff
@@ -0,0 +1,58 @@
+import os
+
+from openvino.tools.mo.back.ie_ir_ver_2.emitter import append_ir_info
+from openvino.tools.mo.utils.cli_parser import get_meta_info
+from openvino.tools.mo.utils.error import Error
+from openvino.tools.mo.utils.telemetry_utils import get_tid
+from openvino.tools.mo.utils.utils import refer_to_faq_msg
+from openvino.tools.mo.utils.version import get_simplified_mo_version
+
+try:
+    import openvino_telemetry as tm
+except ImportError:
+    import openvino.tools.mo.utils.telemetry_stub as tm
+
+
+def serialize(ngraph_function, xml_path, argv=None):
+    """
+    Serializes nGraph function to IR.
+    :param ngraph_function: nGraph function to serialze.
+    :xml_path: path to IR.
+    :argv: Command line arguments info which will be appended to IR.
+    """
+    from openvino.runtime import serialize  # pylint: disable=import-error,no-name-in-module
+    telemetry = tm.Telemetry(tid=get_tid(), app_name='Model Optimizer', app_version=get_simplified_mo_version())
+    telemetry.start_session('mo')
+    telemetry.send_event('mo', 'version', get_simplified_mo_version())
```

Comment:
So, the telemetry looks to be the only reason to keep `mo.serialize` separate from `ov.serialize`. But why is it useful? The most important information comes from `mo.convert`. @jane-intel, @rkazants, @andrei-kochin, please share your opinion.

Reply:
Telemetry is used here for sending error info during creating of output directory. This information is not very useful.

Reply:
As the telemetry in `mo.serialize` doesn't add any real value then there is no reason to have `mo.serialize` as something separate from `ov.serialize`, we will have difficulties to explain to user why one should call `mo.serialize` instesd of `ov.serialize`. An extra import is not a big deal here, I think.

Reply:
mo.serialize() removed.